website/docs/r/ui_plugin.html.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_ui_plugin"
+sidebar_current: "docs-vcd-resource-ui-plugin"
+description: |-
+  Provides a VMware Cloud Director UI Plugin resource. This can be used to manage UI Plugins.
+---
+
+# vcd\_ui\_plugin
+
+Provides a VMware Cloud Director UI Plugin resource. This can be used to manage UI Plugins in VCD, for example to add a new
+plugin from an local ZIP file, to publish/unpublish a UI Plugin to different Organizations, etc.
+
+-> Creating, updating and deleting UI Plugins requires System administrator privileges.
```

Comment:
Nit. We're usually capitalizing this in notes:

```suggestion
-> Creating, updating and deleting UI Plugins requires System Administrator privileges.
```

Reply:
Fixed!